{{cookiecutter.repo_name}}/install_fedora_dependencies.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+OS_REQUIREMENTS_FILENAME="requirements.yum"
+
+# Handle call with wrong command
+function wrong_command()
+{
+  echo "${0##*/} - unknown command: '${1}'"
+  usage_message
+}
+
+# Print help / script usage
+function usage_message()
+{
+  echo "usage: ./${0##*/} <command>"
+  echo "available commands are:"
+  echo -e "\tlist\t\tPrint a list of all packages defined on ${OS_REQUIREMENTS_FILENAME} file"
+  echo -e "\thelp\t\tPrint this help"
+  echo -e "\n\tCommands that require superuser permission:"
+  echo -e "\tinstall\t\tInstall packages defined on ${OS_REQUIREMENTS_FILENAME} file. Note: This\n\t\t\t   does not upgrade the packages already installed for new\n\t\t\t   versions, even if new version is available in the repository."
+  echo -e "\tupgrade\t\tSame that install, but upgrate the already installed packages,\n\t\t\t   if new version is available."
+
+}
+
+# Read the requirements.apt file, and remove comments and blank lines
+function list_packages(){
+     cat ${OS_REQUIREMENTS_FILENAME} | grep -v "#" | grep -v "^$";
+}
+
+function install()
+{
+    list_packages | xargs yum -y install;
```

Comment:
`yum install` command must explicitly not update a package if it is already installed